### PR TITLE
Support relations in group controls

### PR DESCRIPTION
**Pull request by mournfulCoroner**
Related to https://github.com/datalens-tech/datalens/issues/89

- set up the correct operation of links in group selectors (include migration links from old selectors to new)
- add the ability to select the current item in the group selector relations dialog
- a new way of displaying debug info for group selectors
- new system for counting loaded selectors in group selectors
- change naming of subitems in groups from items to group

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "@gravity-ui/app-layout": "^1.7.0",
     "@gravity-ui/browserslist-config": "^4.3.0",
     "@gravity-ui/chartkit": "^4.22.1",
+    "@gravity-ui/dashkit": "^6.9.0",
```
Comment (mournfulCoroner) [inline package.json:56]: need fix

File: src/ui/units/dash/containers/Dialogs/DialogRelations/hooks/helpers.ts [hunk 2]
```diff
@@ -463,17 +463,20 @@ export const getRelationsData = ({
...
+
     const metaItems = getCurrentNamespaceItems({
         metaItems: metaData,
-        widgetId: widgetMeta.widgetId,
+        id: currentId,
     });
 
     const relationsItems: Array<DashkitMetaDataItem> = [];
 
     // filtering logic is reproduced from old links
```
Comment (mournfulCoroner) [inline src/ui/units/dash/containers/Dialogs/DialogRelations/hooks/helpers.ts:475]: can be deleted

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9094-support-relations-in-group-controls-749/component-dialog-relations-view-749/ru/">component.dialog-relations.view (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

File: src/i18n-keysets/component.dialog-relations.view/en.json [hunk 3]
```diff
@@ -26,6 +26,7 @@
   "label_charts": "charts",
   "label_choose-field-for-alias": "Choose field for alias",
   "label_choose-fields-from-list": "Choose fields from list",
+  "label_current-selector": "Selector: ",
```
Comment (Marginy605) [inline src/i18n-keysets/component.dialog-relations.view/en.json:29]: It's a bad practice to add space and colon inside keysets! Need to remove it and add it on usage

**Review by Marginy605 — CHANGES_REQUESTED** [review]

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9094-support-relations-in-group-controls-749/component-dialog-relations-view-749/en/">component.dialog-relations.view (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Review by Marginy605 — CHANGES_REQUESTED** [review]

**Review by flops — APPROVED** [review]

**Review by Marginy605 — APPROVED** [review]

**Pull request merged**